### PR TITLE
Need to handle type checks

### PR DESCRIPTION
**Pull request by danfu-phyos**
Need to handle type checks, possibly in mapping.py

### CODE REVIEW

**Comment by danfu-phyos** [comment]
Not sure how we want to implement type checks in the repo -- we could use assert isinstance(my_var,class) or we could use a try: except block or some other method

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Python CircleCI 2.0 configuration file
```
Comment (phyos-engineering) [inline .circleci/config.yml:1]: Motivations?

File: calculator.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"""
```
Comment (phyos-engineering) [inline calculator.py:1]: I think "git add . " accidentally added this. We can delete this.

File: test_calculator.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+"""
```
Comment (phyos-engineering) [inline test_calculator.py:1]: Same as above. We can delete this from repo.

**Review by phyos-engineering — CHANGES_REQUESTED** [review]

**Review by phyos-engineering — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (phyos-engineering) [reply .circleci/config.yml:1]: Wait, just saw a red flag. It seems this commit overwrote cov=mapping with cov=calculator which I'm guessing wasn't intentional

**Pull request merged**